source/type/vehicle_journey.h

```diff
@@ -35,6 +35,7 @@ www.navitia.io
 #include "validity_pattern.h"
 #include <set>
 #include <boost/serialization/split_member.hpp>
+#include "routing/journey_pattern_container.h"
```

Comment:
Not sure VJ should know about JP and JPP (used for `get_corresponding_stop_time()`).
Any opinion/idea welcome on that.

Reply:
I have no opinion about except that you have to include header when only it is necessary. 
I htink you know that. The compiler can say it if that is missing :robot:

Reply:
OK, for the forward, but the main question is "Between VJ and JPP, who should know about the other?"

Reply:
Idealy `::type` should have no dependency to most of our codebase, and certainly not to `::routing`